python/python-core/python-functions/calling-functions.md

```diff
@@ -40,7 +40,7 @@ Gives the following output:
 Function time!
 ```
 
-A parameter in Python is a variable which can be passed into functions so that methods can be performed onto it.
+A parameter in Python is a variable that can be passed into functions so that methods can be performed onto it.
```

Comment:
We can do more than methods on any value so I'd maybe remove that part? Also, parameters aren't technically passed in, it's values for those parameters. i.e. the arguments, that are passed in. 

```suggestion
A parameter in Python is a variable that represents a value passed into a function.
```